addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightStripeHandler.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yeelight.handler;
+
+import com.yeelight.sdk.device.ConnectState;
+import com.yeelight.sdk.device.DeviceStatus;
+import com.yeelight.sdk.services.DeviceManager;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.yeelight.YeelightBindingConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link YeelightStripeHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Coaster Li (lixin@yeelink.net) - Initial contribution
```

Comment:
The email is not required, can you do a quick search and replace to remove them all